src/core/linter-rules/warn-broken-refs.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Linter rule "warn-local-ref".
+ * Warns about href's that link to nonexistent id's in a spec
+ */
+import { lang as defaultLang } from "core/l10n";
+import LinterRule from "core/LinterRule";
+
+const name = "warn-broken-refs";
+
+const meta = {
+  en: {
+    description: "Broken local reference found in document.",
+    howToFix: "Please fix the href attributes of the links mentioned.",
```

Comment:
"fix the href attributes" might not be possible, because it might be ReSpec creating the href.

Reply:
Maybe, "fix the links mentioned" instead,